### PR TITLE
1.13 Marathon bump 1.8.222

### PR DESCRIPTION
**Pull request by ANeumann82**
## High-level description

Bump Marathon to 1.8.222 
This is mostly to include the change for MARATHON-8681
What features does this change enable? What bugs does this change fix?

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5460](https://jira.mesosphere.com/browse/DCOS_OSS-5460) Bump Marathon to 1.8.222 on 1.13
- MARATHON-7977: Clarify support for ranges and sets with constraint operators
- DCOS-51846: Add seccomp integration tests
- MARATHON-8627: Revive and suppress offers based on instance state
- MARATHON-8685: Exit with 111 if Marathon could not bind to address
- MARATHON-8660: Added maintenance mode to info endpoint
- MARATHON-8681: Remove strict validation of external volume name

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [compare on github](https://github.com/mesosphere/marathon/compare/9f3550487...86475ddac)
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [x] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by ANeumann82** [comment]
@mesosphere-mergebot Label Ready for Review

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +18,16 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Prune VIPs with no backends in order to avoid unbounded growth of state and messages exchanged among `dcos-net` processes. (DCOS_OSS-5356)
 
+* [Marathon] Clarify support for ranges and sets with constraint operators (MARATHON-7977)
```
Comment (timcharper) [inline CHANGES.md]: omit

File: CHANGES.md [hunk 2]
```diff
@@ -16,6 +18,16 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
...
+
+* [Marathon] Revive and suppress offers based on instance state (MARATHON-8627)
```
Comment (timcharper) [inline CHANGES.md]: omit

**Comment by ANeumann82** [comment]
@mesosphere-mergebot sync

**Pull request merged**